docs/developer/soffits/README.md

```diff
@@ -0,0 +1,46 @@
+# uPortal Soffits
```

Comment:
I do hope we can have modules in the portal that are not Soffits.

My issue with the name "Soffit" is that it obfuscates what these are. "Underside of an arch" doesn't help me understand what these are. They might be tunnels, or tubes, or tentacles, or puppet strings, or WSRP-but-no-SOAP. 

Maybe URP "uPortal Remote Portlets"? Conceptually that's what this is, it's WSRP except not handcuffed to a dead-on-arrival formal specification and instead ad-hoc inventing what uPortal might need. URC "uPortal Remote Channels" to escape the overloaded "portlet" word? (I knew Channels would rise again.)

The linked naming discussion I thought was pretty good before it went off the rails with the "microservices" reference.